mergify_engine/tests/functional/test_simulator.py

```diff
@@ -122,7 +122,8 @@ async def test_simulator_with_token(self):
         assert r.json() == {
             "errors": [
                 "expected str @ pull_request_rules → item 0 → actions → label → remove → item 0",
-                "expected str @ pull_request_rules → item 0 → conditions → item 0",
+                "extra keys not allowed @ pull_request_rules → item 0 → conditions → item 0 → -conflict",
+                "required key not provided @ pull_request_rules → item 0 → conditions → item 0 → Any('or', 'and', msg=None)",
```

Comment:
that last one is kinda weird, can we have a better string for the last element?

Reply:
I found a way to only display the extra keys message, which is enough to understand that `-conflict:` is invalid.